PocketKit/Sources/Analytics/Events/System.swift

```diff
@@ -11,17 +11,24 @@ public struct System: Event, CustomStringConvertible {
     public static let schema = "iglu:com.pocket/system_log/jsonschema/1-0-0"
 
     let type: System.SystemLogType
-    let source: MigrationSource
+    let source: MigrationSource!
```

Comment:
Why is this implicitly unwrapped? If it's only used under _some_ conditions, perhaps it can be marked as optional, or refactored such that it's extracted from here, and maybe used as an associated value in enum cases where it's needed.

TL;DR: I'm scared of implicitly-unwrapped optionals, especially if we're saying `nil` is a valid case with one of the initializers.

Reply:
This does need a refactor, I figured I'd get away with it here.
I made it implicit for convenience. If we're doing migration stuff we'll init it, if not we don't use it.